examples/WorkstationWriteBasicCookstyle.md

```diff
@@ -187,10 +187,25 @@ The actual matching happens here. The node matching block defined in the previou
   end
 end
 ```
-The last method handles the autocorrection portion of the Cookstyle rules, and replaces the node with the defined value. 
+The last method handles the autocorrection portion of the Cookstyle rules, and replaces the node with the defined value.
+
+## Testing Example Rule in Cookbook
+To test example rule, a `.rubocop.yml` needs to be created directly in the cookbook or as a reference, inherited within the cookbook.
```

Comment:
That's awesome! Future consideration: Getting the Rule in the Pipeline. But, that's for another PR.